src/components/panel/panel.js

```diff
@@ -335,7 +335,7 @@ angular
  * Overlapping the panel with an element:
  * `new MdPanelPosition()
  *     .relativeTo(someElement)
- *     .withPanelXPosition('align-left')
+ *     .withPanelXPosition('align-start')
```

Comment:
Consider having these options defined and accessible for the user, similar to the $mdPanel.animation values.